WordPress/src/main/res/layout/fragment_login_no_sites_empty_view.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:fillViewport="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/margin_extra_large"
+        android:layout_marginStart="@dimen/margin_extra_large"
+        android:layout_marginTop="@dimen/margin_extra_extra_large"
+        android:gravity="center|center_horizontal|center_vertical">
+
+        <androidx.constraintlayout.widget.Group
+            android:id="@+id/login_empty_view_user_section"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:visibility="gone"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:constraint_referenced_ids="image_avatar,text_username, text_displayname"
+            tools:visibility="visible"/>
+
+        <ImageView
+            android:id="@+id/image_avatar"
+            android:layout_width="@dimen/login_empty_view_avatar_width"
+            android:layout_height="@dimen/login_empty_view_avatar_height"
+            android:layout_marginBottom="@dimen/login_empty_view_avatar_margin_bottom"
+            android:contentDescription="@string/login_empty_view_avatar_content_description"
+            app:layout_constraintBottom_toTopOf="@+id/text_displayname"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:srcCompat="@drawable/ic_user_white_24dp" />
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/text_username"
+            style="@style/Login.EmptyView.TextView.Username"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingBottom="@dimen/margin_extra_medium_large"
+            app:layout_constraintBottom_toTopOf="@+id/login_empty_view_illustration"
+            app:layout_constraintStart_toStartOf="@+id/text_displayname"
+            app:layout_constraintTop_toBottomOf="@+id/text_displayname"
+            tools:text="\@droidtester2021" />
+
+        <com.google.android.material.textview.MaterialTextView
+            android:id="@+id/text_displayname"
```

Comment:
1) Minor (🔍): I suggest reordering this view and moving that on top of the `text_username`, code wise, just to keep the design order so that it makes it easier to read through. I was confused initially while reading this layout file.
2) Minor (🔍): I suggest renaming the `android:id` to `text_display_name` as the current naming results in a typo.
3) Question (❓): The design specifies that there should exist a `4dp` bottom margin. Am I missing something?

Reply:
> 1. Minor (🔍): I suggest reordering this view and moving that on top of the `text_username`, code wise, just to keep the design order so that it makes it easier to read through. I was confused initially while reading this layout file.

Based on this comment, I let AS reorder the entire view. Thought you were confused before. I've committed it because I want you to see how AS believes it should be ordered.

> 3. Question (❓): The design specifies that there should exist a `4dp` bottom margin. Am I missing something?

@ParaskP7  - there were no specs for this view ... so this is based this off of Woo and then adjusted to match what Chris and I discussed. Where are you seeing a spec sheet for this?

Reply:
> Based on this comment, I let AS reorder the entire view. Thought you were confused before. I've committed it because I want you to see how AS believes it should be ordered.

Yeap, AS can reorder those in a totally weird way, I have experienced that in the past and still I am not sure why it does that. I understand reformatting it, but reordering should happen like that. 🤔 

> @ParaskP7 - there were no specs for this view ... so this is based this off of Woo and then adjusted to match what Chris and I discussed. Where are you seeing a spec sheet for this?

I see, thanks for explaining that for me. PS: I was checking the specs on Zeplin.

Reply:
@zwarm I think reformatting using AS made it actually worse: 9a905ad28316033fb892c8bcb839186e2cc3827e 😄

Reply:
@ParaskP7 yes, letting AS reformat made it worse - which is why I committed it - just to show you. :)

Reply:
I pushed the reformatted file in 9e49217. Yay

Reply:
> @ParaskP7 yes, letting AS reformat made it worse - which is why I committed it - just to show you. :)

😅